### PR TITLE
Add FR

### PR DESCRIPTION
**Pull request by MarchWorks**

### CODE REVIEW

File: locales/fr.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "home.description": "Générer très facilement divers liens de partage",
+  "home.description2": "Pas de pubs, Pas de suivi, Simple",
```
Comment (nzws) [inline locales/fr.json:3]: Thank you for your contribution!
I don't understand French, but I checked with a machine translation.
Does `de suivi` mean `tracking` (e.g. Google Analytics)?
I was curious about this. I'm sorry if it's right!

Comment (MarchWorks) [reply locales/fr.json:3]: `suivi` means `tracking`. `de` is a french preposition

Comment (MarchWorks) [reply locales/fr.json:3]: `Traçage` may be a better word to use here. I will check it when I wake up

Comment (MarchWorks) [reply locales/fr.json:3]: Ok I checked the web and it seems that `suivi` is the word used (Like for Google Analytics they say `code de suivi Google Analytics`)

Comment (nzws) [reply locales/fr.json:3]: OK, please mention when you're done! I'll merge it.

**Comment by MarchWorks** [comment]
You're welcome

**Pull request merged**